Sources/TuistKit/Commands/CleanCommand.swift

```diff
@@ -9,7 +10,12 @@ struct CleanCommand: ParsableCommand {
         )
     }
 
+    @Argument(help: "The cache categories to be cleaned. If no category is specified, the whole cache is cleaned.")
```

Comment:
Is this the desired default? Should we clear everything? Some tools like cocoapods clean just certain parts and then allow specifying clearing everything, just wondering what would be the best here

Reply:
For sure should be like this until 2.0

Reply:
as a default, I'd probably clean everything related to the current project. We'd have to restructure a bit the internal structure to do this efficiently but it imho makes the most sense.